apps/foundation/pages/club/index.tsx

```diff
@@ -28,8 +28,15 @@ export const getStaticProps: GetStaticProps = async context => {
     // So, we need to keep the same order of the `queryKey` array
 
     // queryKey: [arts, locale, searchTerm, category, page]
-    queryKey: ['arts', locale, '', null, '1'],
-    queryFn: () => getArts({ locale: locale as StrapiLocale }),
+    queryKey: ['arts', locale, null, null, '1'],
+    queryFn: () =>
+      getArts({
+        populate: ['artist.user.avatar', 'categories', 'images', 'likers'],
```

Comment:
As you might have already noticed, Art-Artist relation is now directly related to the User

```suggestion
        populate: ['artist.avatar', 'categories', 'images', 'likers'],
```